packages/docs/README.md

```diff
@@ -1,26 +1,60 @@
-# ZeppelinOS documentation site
+# ZeppelinOS documentation website
 
-Centralized documentation site for ZeppelinOS.
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg)](https://github.com/RichardLitt/standard-readme)
 
-To view this website locally, clone the repository with:
+> Documentation for the ZeppelinOS smart contract platform.
 
-`git clone https://github.com/zeppelinos/zos-docs.git`
+## Development setup
+
+First, install [Node.js](http://nodejs.org/), [npm](https://npmjs.com/) and
+[git](https://git-scm.com/).
+
+Then, clone the repository:
+
+```sh
+git clone https://github.com/zeppelinos/zos-docs.git
```

Comment:
shouldn't we ask the user to clone the main `zos` repo and navigate to `packages/docs` instead?
```
git clone https://github.com/zeppelinos/zos.git
cd ./packages/docs
```

Reply:
@elopio IMO this step needs to be fixed since we are not using that repo anymore